src/azure-cli-core/setup.py

```diff
@@ -107,8 +105,10 @@
         ":python_version<'2.7.9'": ['pyopenssl', 'ndg-httpsclient', 'pyasn1'],
         ':python_version<"3.0"': ['futures'],
         ":python_version<'3.0'": ['antlr4-python2-runtime'],
-        ":python_version>='3.0'": ['antlr4-python3-runtime']
+        ":python_version>='3.0'": ['antlr4-python3-runtime'],
+        "test": TESTS_REQUIRE
```

Comment:
nit: mind adding a comma here so that future diffs don't have the extra line noted as changed? Sorry, personal pet peeve that you had no control over.